cursor/README.md

```diff
@@ -0,0 +1,35 @@
+---
+display_name: Cursor IDE
+description: Add a one-click button to launch Cursor IDE
+icon: ../.icons/cursor.svg
+maintainer_github: coder
+verified: true
+tags: [ide, cursor, helper]
+---
+
+# Cursor IDE
+
+Add a button to open any workspace with a single click in Cursor IDE.
```

Comment:
"Requires the local Coder VSCode extension. You can [import this extension into Cursor](https://docs.cursor.com/get-started/migrate-from-vscode)."

Reply:
For me it auto prompts to install the extension if not installed already (Similar to VS Code).
![image](https://github.com/user-attachments/assets/adc20f7e-eb75-49aa-a868-0937f87bbac1)